linux_os/guide/system/software/integrity/crypto/openssl_use_strong_entropy/bash/shared.sh

```diff
@@ -0,0 +1,5 @@
+# platform = Red Hat Enterprise Linux 8
+
+cat > /etc/profile.d/cc-config.sh <<- 'EOM'
```

Comment:
I would call it `openssl-rand.sh` or something like that. `cc-config.sh` seems like something that could conflict with some other rule / logic.